src/main/java/apoc/convert/Convert.java

```diff
@@ -79,12 +88,91 @@ else if (list.getClass().isArray() && !list.getClass().getComponentType().isPrim
         }
         return null;
     }
-
+    
     @SuppressWarnings("unchecked")
+    private <T> List<T> convertToList(Object list, Class<T> type) {
+        List<Object> convertedList = convertToList(list);
+        if (convertedList == null) {
+        	return null;
+        }
+        Stream<T> stream = null;
+    	if (type.isAssignableFrom(Long.class)) {
+    		final Exact exact = new Exact();
+    		stream = (Stream<T>) convertedList.stream().map(e -> {
+    			if (e == null) {
```

Comment:
I found in the Util class the [toLong](https://github.com/neo4j-contrib/neo4j-apoc-procedures/blob/3.2/src/main/java/apoc/util/Util.java#L219) method maybe i wrap it in private method in the Covert class, something like this:
```java
    private Long toLong(Object value) {
    	try {
    		return Util.toLong(value);
    	} catch (NullPointerException e) {
    		return null;
    	}
    }
```

Or i can change the method in Util class in something like this:
```java
(-) public static long toLong(Object value) {
(+) public static Long toLong(Object value) {
(+)     if (value == null) return null;        
        if (value instanceof Number) return ((Number)value).longValue();
        return Long.parseLong(value.toString());
    }
```
Or completly change the method as you suggested:

```java
public static Long toLong(Object value) {
   try {
      return Long.parseLong(value.toString());
   } catch (NumberFormatException e) {
      return null;
   }
}
```

What do you think?